client/settings/payments-and-transactions-section/__tests__/index.test.js

```diff
@@ -82,4 +82,37 @@ describe( 'PaymentsAndTransactionsSection', () => {
 			'WOOTESTING'
 		);
 	} );
+
+	it( 'shows the full bank statement preview', () => {
+		const updateAccountStatementDescriptor = jest.fn();
+		const mockValue = 'WOOTESTING, LTD';
+		useAccountStatementDescriptor.mockReturnValue( [
+			mockValue,
+			updateAccountStatementDescriptor,
+		] );
+		render( <PaymentsAndTransactionsSection /> );
+
+		expect(
+			document.querySelector(
+				'.full-bank-statement .transaction-detail.description'
+			)
+		).toHaveTextContent( mockValue );
+	} );
+
+	it( 'shows the shortened customer bank statement preview', () => {
```

Comment:
It would be good if we assert both visible and hidden cases here based on the value of `isShortAccountStatementEnabled`

Reply:
I couldn't get `fireEvent.click( screen.getByLabelText( 'Add customer order number to the bank statement' ) )` to trigger the state change...I'm not sure if that _should_ work or not. Instead, I added another test to ensure that the 2nd preview box is _not_ shown if that box is not checked.